components/fctl/pkg/metadata.go

```diff
@@ -21,19 +22,15 @@ func ParseMetadata(array []string) (map[string]any, error) {
 	return metadata, nil
 }
 
-func PrintMetadata(out io.Writer, metadata map[string]any) error {
+func PrintMetadata(out io.Writer, metadata map[string]string) error {
```

Comment:
Why not use metadata.Metadata instead of map[string]string directly ?

Reply:
You're right, i will change